docs/run-move-locally.md

```diff
@@ -0,0 +1,278 @@
+
+```
+libra% dev publish 0 /var/folders/tq/8gxrrmhx16376zxd5r4h9hhn_x1zq3/T/.tmpigAZCx
+
+waiting .....transaction is stored!
+no events emitted.
+Successfully published module
+```
+Upon successful execution of the `dev publish` command, the bytecode for `MyModule` is published under the sender’s account. To use the procedures and types declared in `MyModule`, other transaction scripts and modules can import it using `import <sender_address>.MyModule`.
+
+ Subsequent modules published under `<sender_address>` must not be named `MyModule`. Each account may hold at most one module with a given name. Attempting to publish a second module named `MyModule` under `<sender_address>` will result in a failed transaction.
+
+## Compile and Execute Transaction Scripts
+
+### Create Transaction Script
+
+<blockquote class="block_note">
+
+**Note**: you'll find samples of transaction scripts in the `libra/language/stdlib/transaction_scripts` directory.
+</blockquote>
+
+Now let’s write the following script to use `MyModule` and save it as `custom_script.mvir`:
+
+```
+import 0x0.LibraAccount;
+import 0x0.LibraCoin;
+import {{sender}}.MyModule;
+
+main(amount: u64) {
+  let coin: R#LibraCoin.T;
+  coin = LibraAccount.withdraw_from_sender(move(amount));
+  //calls the id procedure defined in our custom module
+  LibraAccount.deposit(get_txn_sender(), MyModule.id(move(coin)));
+  return;
+}
+```
+
+In this script, `{{sender}}`will be automatically replaced with sender account address when the script is executed. Alternatively, you could write an import with a fully qualified address:
+
+```
+ import 0x810abcc08dbed34ea15d7eb261b8001da6a62d72acdbf87714dd243a175f9b62.MyModule;
+```
+
+### Compile Transaction Script
+
+To compile your transaction script, use the [dev compile](reference/libra-cli#dev-d-mdash-operations-related-to-move-transaction-scripts-and-modules) command.
+
+```
+libra% dev compile 0 <path_to_custom_script.mvir> script
+```
+
+ `custom_script.mvir` is in Move IR, and upon successful compilation of `custom_script.mvir` the compiler will  output the corresponding bytecode file. You'll use this bytecode file (not the `.mvir` file) when you execute this script. After the script is successfully compiled, you'll see the path to the bytecode file in your output:
+
+```
+Successfully compiled a program at /var/folders/tq/8gxrrmhx16376zxd5r4h9hhn_x1zq3/T/.tmpDZhL21
+```
+
+If you would like to save the bytecode file at a specific path, and not at a temporary path, you can specify it as an argument to the [dev compile](reference/libra-cli#dev-d-mdash-operations-related-to-move-transaction-scripts-and-modules) command.
+
+### Execute Transaction Script
+
+To execute your script, use the [dev execute](reference/libra-cli#dev-d-mdash-operations-related-to-move-transaction-scripts-and-modules) command on the bytecode output from [Compile the Transaction Script](#compile-the-transaction-script) step above.
```

Comment:
Similar here: 
```suggestion
To execute your script, use the [dev execute](reference/libra-cli#dev-d-mdash-operations-related-to-move-transaction-scripts-and-modules) command on the bytecode output from [Compile Transaction Script](#compile-transaction-script) step above.
```

Reply:
Fixed the anchor link and updated the PR.